core/platform/datastore/cloud_datastore_services.py

```diff
@@ -146,14 +146,12 @@ def delete_multi(keys):
     return ndb.delete_multi(keys)
 
 
-def query_everything(**kwargs):
-    # type: (**Dict[str, Any]) -> ndb.Query
+def query_everything(**kwargs: Dict[str, Any]) -> ndb.Query:
```

Comment:
I think the type of kwargs here should be Any right?
See [here](https://github.com/oppia/oppia/blob/aefe95731d353fe7142f8b18343e49c81e39adcf/jobs/io/ndb_io_test.py#L37).